package.json

```diff
@@ -78,6 +83,7 @@
     "targz": "1.0.1",
     "uglify-js": "2.6.4",
     "underscore": "1.8.3",
-    "watch": "0.19.1"
+    "watch": "0.19.1",
+    "webpack": "1.14.0"
```

Comment:
Yes, and thats great news 🎉  
Although, until we finally drop ourself support for node 0.10&co I will suggest we stick to v1. I'll prepare a v2 upgrade on a separate PR, what you think?